components/webdriver_server/lib.rs

```diff
@@ -213,10 +230,15 @@ struct Handler {
 
     session: Option<WebDriverSession>,
 
+    /// Embedder proxy
+    embedder_proxy: EmbedderProxy,
+
```

Comment:
If you follow the suggestion above:

```suggestion
    /// A [`Sender`] that sends messages to the embedder that this `WebDriver instance controls.
    /// In addition to sending a message, we must always wake up the embedder's event loop so it
    /// knows that more messages are available for processing.
    embedder_sender: Sender<WebDriverCommandMsg>,

    /// An [`EventLoopWaker`] which is used to wake up the embedder event loop.
    event_loop_waker: Box<dyn EventLoopWaker>,
```